README.md

```diff
@@ -29,10 +29,10 @@ There are many projects that you can use and contribute to, some of which are li
 
 - [.NET Core (dotnet/core)](https://github.com/dotnet/core)
 - [.NET Core docs (dotnet/docs)](https://github.com/dotnet/docs)
-- [ASP.NET Core (aspnet/home)](https://github.com/aspnet/home)
-- [ASP.NET Core docs (aspnet/Docs)](https://github.com/aspnet/Docs)
+- [ASP.NET Core (dotnet/aspnetcore)](https://github.com/dotnet/aspnetcore)
+- [ASP.NET Core docs (aspnet/AspNetCore.Docs)](https://github.com/aspnet/AspNetCore.Docs)
```

Comment:
@Pilchie the case looks inconsistent

Reply:
@RussKie, The repositories in dotnet organization are all lower-case letters.

Reply:
Yes, this is correct for now.  Eventually AspNetCore.Docs will also move into dotnet, and I assume be lowercased.